scss/input/_layout.scss

```diff
@@ -5,38 +5,47 @@
     $textbox-shadow: 0 2px 2px 1px rgba(0, 0, 0, .06) !default;
     $fieldset-margin: 30px;
 
-    .k-textbox > input,
-    .k-input,
-    .k-textbox {
+    .k-textbox,
+    .k-maskedtextbox {
+        @include border-radius( $border-radius );
+        padding: $input-padding-y $input-padding-x;
+        width: 12.4em;
```

Comment:
Seems like these should be together, or at least indicate that they share the logic. Perhaps we could use a placeholder / mixin for all these, so that the selectors are built based on the usage?